src/Compilers/VisualBasic/Portable/Errors/LocalizableErrorArgument.vb

```diff
@@ -29,7 +34,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' </summary>
         ''' <param name="format">A string to use for formatting.</param>
         ''' <param name="formatProvider">An object that supplies culture-specific format information about format.</param>
-        Public Function ToString_IFormattable(format As String, formatProvider As IFormatProvider) As String Implements IFormattable.ToString
+        Public Overloads Function ToString(format As String, formatProvider As IFormatProvider) As String Implements IFormattable.ToString
```

Comment:
Lets bring that up with compat council.  Would like to handle fixing this bug and the API implications separately.